cmd/file_lint.go

```diff
@@ -20,6 +22,7 @@ var (
 	cmdLintFailSeverity   string
 	cmdLintOutputFilename string
 	cmdLintOnlyFailures   bool
+	cmdCustomFunctionsPath string
```

Comment:
please use `cmdLint` as prefix, those variables are global in decK, so must minimize probability of colliding with other command options.

Reply:
lower case vars are not exported outside the package, however this is visible across the `cmd` package so I agree we should follow the suggested and current prefix pattern.

Reply:
Thanks @rspurgeon this should have been addressed in the latest commit